modules/ROOT/pages/setup-payload-storage-API.adoc

```diff
@@ -1,29 +1,33 @@
 = Payload Storage API for Partner Manager
 
-Anypoint Partner Manager enables you to configure your own storage solution, such as database, Amazon S3, Azure Blob Storage, and so on to store transmission content, such as incoming and outgoing B2B message payloads.  
+Anypoint Partner Manager enables you to configure your own storage solution (for example, a database, Amazon S3, or Azure Blob Storage) to store transmission content, such as incoming and outgoing B2B message payloads.
 
-Before you create your first trading partner and message flow, you must configure your payload storage API within the host organization in Partner Manager.
+Before you create your first xref:configure-partner#create-partner[trading partner] and xref:configure-message-flow[message flow], you must configure your payload storage API within the host organization in Partner Manager.
 
-To store your content, you must implement an API that provides Anypoint Partner Manager with a standardized way to connect to your unique storage.
+To use your own storage solution for warehousing content, you must implement an API that provides Anypoint Partner Manager with a standardized way to connect to your storage.
```

Comment:
"storage solution for warehousing content"?
And do we need to explain what an API is?

Reply:
Changed to "to use your own storage solution" to match the intro paragraph. 


MuleSoft users will know the definition of API, so that explanation is not required.